Flutter User Facing API.dart

```diff
@@ -0,0 +1,71 @@
+class BleManager {
```

Comment:
CentralManager is a better name if we are not obligated to keep backward compatibility as in RN.

Reply:
So, in the end, it would be nice to have an API in the form of BluetoothManager which could create "CentralManager" and "PeripheralManager" instances and also contain functions such as enableRadio().